### DISCUSSION TITLE
915 MHz ot-efr32

### DISCUSSION THREAD

**Discussion by kmfnjtze8888** [openthread/openthread#8786] [Q&A]
Hello,

Is 915 MHz for ot-efr32 no longer an option? I tried to build the latest OpenThread from the ot-efr32 for my brd4170a but it looks like the only place to enable the `RADIO_CONFIG_915MHZ_OQPSK_SUPPORT` is now only in src/src/board_config.h, but the comment on it says _"Dev board doesn't support OQPSK modulation in 915MHz band."_

Also in the src/efr32mg12 directory, the folders `brd4161a`, `brd4166a`, `brd4170a`, `brd4304a` are now gone..

**Comment by silabs-YupingX** [comment]
Hi @kmfnjtze8888, are you aware of the instructions described in [AN1350](https://www.silabs.com/documents/public/application-notes/an1350-openthread-single-band-proprietary-sub-ghz.pdf)? Section 1.1.4 has a list of supported radio boards. BRD4170a is indeed supported but not the other radio boards.

**Reply by kmfnjtze8888** [reply]
Thanks @silabs-YupingX.

Btw, how do I build the binaries for 915 MHz OQPSK from the ot-efr32 repo here on github? I used to be able to do it by changing the values of the macros to `RADIO_CONFIG_915MHZ_OQPSK_SUPPORT 1` and `RADIO_CONFIG_2P4GHZ_OQPSK_SUPPORT 0 `in **./src/efr32mg12/brd4170a/board_config.h** but on the latest commit, it looks like the **./src/efr32mg12/brd4170a** directory is no longer there.

I tried changing those same constants in **./src/src/board_config.h** as it looks like that's the only place where they are defined, but I received an error when I attempted to build it.

**Reply by brian-level** [reply]
I couldn't get the ot-eft32 build working either and couldn't get support to fix it.  What *is* supported it building this in Simplicity Commander.  Follow the instructions in AN1350 and you can get a working solution for the MG12 chip.   The only way to get the build working in ot-efr32 is to rework the make system to hand-configure the rail (and 915mhz support) etc.  Unless you're building a product you're better off using Commander unfortunately.
